util/misc.go

```diff
@@ -141,12 +141,16 @@ const (
 	PerformanceSchemaName = "PERFORMANCE_SCHEMA"
 	// PerformanceSchemaLowerName is the `PERFORMANCE_SCHEMA` database lower name.
 	PerformanceSchemaLowerName = "performance_schema"
+	// MetricSchemaName is the `METRIC_SCHEMA` database name.
+	MetricSchemaName = "METRIC_SCHEMA"
```

Comment:
Why not use a global variable which type is `model.CIStr` .

Reply:
Go does not support struct constants.

Reply:
How about use `var`?

Reply:
done.